node/Cargo.toml

```diff
@@ -112,3 +112,15 @@ polkadot-test-service = { git = "https://github.com/paritytech/polkadot", branch
 
 # Moonbeam Dependencies
 parachain-staking = { git = "https://github.com/OAK-Foundation/moonbeam", default-features = false, branch = "oak-polkadot-v0.9.20" }
+
+[package.metadata.deb]
```

Comment:
Any value in adding any of the systemd specific config units?

Reply:
I didn't see these configuration options so...there might be?